VAMobile/src/screens/BenefitsScreen/ClaimsScreen/ClaimsHistoryScreen/ClaimsHistoryScreen.tsx

```diff
@@ -122,18 +122,24 @@ function ClaimsHistoryScreen({ navigation }: IClaimsHistoryScreen) {
       backLabelOnPress={navigation.goBack}
       title={title}
       testID="claimsHistoryID">
-      {loadingClaimsAndAppealsList || loadingUserAuthorizedServices ? (
+      {!claimsNotInDowntime && !appealsNotInDowntime ? (
+        <ErrorComponent screenID={ScreenIDTypesConstants.CLAIMS_HISTORY_SCREEN_ID} />
+      ) : loadingClaimsAndAppealsList || loadingUserAuthorizedServices ? (
```

Comment:
Unfortunately it will be needed. In theory you are correct, but if this api fails on the sync screen and it gets to the following screens it would be needed, and primarily if there is an error that is being refreshed. Otherwise what happens is they hit the refresh button and then there is no loading screen before either an error returns or the screen fixes itself randomly. So the loading screen is needed in those instances.